tools/awps-tunnel/client/src/components/api/Parameters.tsx

```diff
@@ -0,0 +1,307 @@
+			.then(() => {
+				setTokenCopied(true);
+				setTimeout(() => {
+					setTokenCopied(false);
+				}, 3000)
+			}).catch(err => console.error("Failed to copy token: ", err));
+	}
+
+	async function sendRequest(): Promise<void> {
+		const token: string = await dataFetcher.invoke("getRestApiToken", url);
+		fetch(url, {
+			method: methodName,
+			headers: {
+				'Content-Type': 'application/json',
+				'Authorization': `Bearer ${token}`
+			},
+			body: JSON.stringify(bodyParameterInputs)
+		}).then(res => {
+			const contentType: string | null = res.headers.get("Content-Type");
+			if (methodName !== "head" && contentType && (contentType.includes("application/json") || contentType.includes("text/json") || contentType.includes("application/problem+json"))) {
+				return res.json();
+			} else {
+				return res;
+			}
+		})
+			.then(res => setResponse(res as APIResponse));
+	}
+
+	const tryIt: React.JSX.Element = <div style={{ flex: 3 }}>
+		<Card className="w-95 m-2">
+			<CardHeader header={<b className="fs-6">Try It</b>} />
+			<CardPreview className="d-flex flex-column align-items-start p-3">
+				<div className="d-flex justify-content-between w-100">
+					<Label>HTTP URL</Label>
+				</div>
+				<Input className="d-inline-flex" readOnly={true}
+					contentAfter={<Icon iconName={urlCopied ? "checkmark" : "copy"} style={{ cursor: "pointer" }}
+						onClick={copyUrl} />}
+					value={url} />
+				<div><Label><b>Headers</b></Label></div>
+				<div className="d-flex justify-content-between w-100">
+					<Label>Authorization</Label>
+				</div>
+				<Input className="d-inline-flex" readOnly={true}
+					contentAfter={<Icon iconName={tokenCopied ? "checkmark" : "copy"} style={{ cursor: "pointer" }}
+						onClick={copyHeaderToken} />}
+					value={token} type={tokenVisible ? "text" : "password"}
+					onClick={() => setTokenVisible(!tokenVisible)} />
+				<div className="d-flex">
+
+				</div>
+				{requestParameters.length > 2 && <div><Label><b>Parameters</b></Label></div>}
+				{requestParameters.length > 2 && requestParameters.map(({
+					parameter,
+					inputComponent
+				}, index) => (parameter.name !== "api-version" && parameter.name !== "hub" &&
+					<div key={index}>
+						<div className="d-flex">
+							<Label className="fs-6 me-2">{parameter.name}</Label>
+							{parameter.required && <Label className="text-danger">required</Label>}
+						</div>
+						<div className="d-flex flex-column">
+							{inputComponent}
+							{parameter.description && <small className={"form-text"}>{parameter.description}</small>}
+						</div>
+					</div>
+				))}
+				{requestBody && requestBody.length > 0 && <div><Label><b>Body</b></Label>
+					<div className="d-flex">
+						<Label className="fs-6 me-2">{requestBody[0].name}</Label>{ }
+						{requestBody[0].required && <Label className="text-danger">required</Label>}
+					</div>
+					<JSONInput locale={locale}
+						placeholder={bodyParameterInputs}
```

Comment:
@showPlaceholder(): Expected parameter 'placeholder' either an object or an array
can only pass in the json object, but can't change the display of json with double quotes